lib/install.js

```diff
@@ -2,16 +2,10 @@ const logger = require('@blackbaud/skyux-logger');
 const npmInstall = require('./utils/npm-install');
 const cleanup = require('../lib/cleanup');
 
-const settings = {};
-
-if (logger.logLevel === 'verbose') {
-  settings.stdio = 'inherit';
-}
-
 async function install() {
   try {
     await cleanup.deleteDependencies();
-    await npmInstall(settings);
+    await npmInstall();
```

Comment:
Moved the `stdio` logic to the `npmInstall()` function itself to keep the functionality consistent when called from any other location in the library.